src/raftstore/store/store.rs

```diff
@@ -260,6 +281,11 @@ impl<T: Transport, C: PdClient> Store<T, C> {
         let peer = self.region_peers.get_mut(&region_id).unwrap();
         let timer = SlowTimer::new();
         try!(peer.raft_group.step(msg.take_message()));
+        if let Some(ref k) = k {
+            if peer.raft_group.has_ready() {
```

Comment:
need handling ready here? 
I guess if step snapshot ok, the ready already returns true.